pkg/kn/commands/service/service.go

```diff
@@ -12,13 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package commands
+package service
```

Comment:
Can we have the package named a bit more specific? We already have a "serving" package which deals about shared tasks around serving.

What's about prefixing all commands packages like to `cmd_service`, `cmd_revision`, ... ?

Reply:
Well with `commands` as top level adding `cmd_*` seems redundant...